labs/06-volumes.md

```diff
@@ -8,9 +8,12 @@ So if you need to persist data, do it outside of the containers.
 
 You have two different ways of mounting data from your container `bind mounts` and `volumes`.
 
-**A bind mount** is the simpler one to understand. It takes a host path `/data` and mounts it inside your container eg. `/opt/app/data`. The positive about bind is that it is easy and connects directly to the host filesystem. The downside is that you need to specify it at runtime with no indication on what mounts a given container has, and that you need to deal with backup,migration etc. in an tool outside the Docker ecosystem.
+**A bind mount** is the simpler one to understand. It takes a host path, like `/data`, and mounts it inside your container eg. `/opt/app/data`.
+The good thing about bindmounts is that they are easy and allow you to connect directly to the host filesystem.
```

Comment:
We use `bindmount` as one word in several places. I can't find anywhere else that does this, so we should probably use `bind mounts`

Reply:
Replaced all /bindmount/ with /bind mount/ in 2b4d547